src/lambdas/breeds-get.ts

```diff
@@ -0,0 +1,63 @@
+import fetch from 'node-fetch'
+import * as constants from './shared/constants'
+import { Response } from './shared/types'
+
+interface BreedResponse extends Response {
+  body: string[]
+}
+
+interface DogBreeds {
+  // message type is any because this can either be an object (successful response)
+  // or a string (failure)
+  message: any
+  status: string
+  code: number
+}
+
+// Method to flatten response from Breed API
+// input should be a list of key-value pairs
+// where key = breed-type in string
+// and value = string array of sub-breed types (0 or more)
+const flatten = (data: any): string[] => {
```

Comment:
The instructions state there should be no linting errors, so we should type this more specially than `any`.

Reply:
I've resolved data type to be data: UnflattenedDogBreed

where
interface UnflattenedDogBreed {
  [key: string]: string[]
}